dcap-libs/action.yaml

```diff
@@ -16,7 +16,7 @@ runs:
         source /etc/lsb-release &&
         sudo mkdir -p /etc/apt/keyrings &&
         wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | gpg --dearmor | sudo tee /etc/apt/keyrings/intel-sgx.gpg > /dev/null &&
-        echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/intel-sgx.gpg] https://download.01.org/intel-sgx/sgx_repo/ubuntu $DISTRIB_CODENAME main" | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+        echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/intel-sgx.gpg] https://download.01.org/intel-sgx/sgx_repo/ubuntu $DISTRIB_CODENAME main" | sudo tee /etc/apt/sources.list.d/intel-sgx.list &&
```

Comment:
looks like I'm 0-2 for getting all of the `&&` in the actions :( #5

One can see the failing builds at, https://github.com/mobilecoinfoundation/sgx/actions/runs/5673964066/job/15378114907?pr=379
and the now fixed version at https://github.com/mobilecoinfoundation/sgx/actions/runs/5675892766/job/15381805298?pr=381